attest/core/src/ias/verify.rs

```diff
@@ -228,6 +228,18 @@ impl VerificationReportData {
                 })?;
         Ok(timestamp.into())
     }
+
+    /// Try to get public key bytes from the quote
+    pub fn signing_key_bytes(&self) -> Result<[u8; 32], VerifyError> {
+        let report_data = self.quote.report_body()?.report_data();
+        let report_data_bytes: &[u8] = report_data.as_ref();
+        if report_data_bytes.len() != 64 {
+            return Err(VerifyError::IasQuoteMismatch);
+        }
+        Ok(report_data_bytes[32..64]
+            .try_into()
+            .map_err(EncodingError::from)?)
+    }
```

Comment:
> @eranrund commented
> I don't think this belongs here. The consensus enclave stores its signing key public key on the report data, but it is not guaranteed that each enclave we build will do that (signing key does not even exist for all enclaves we have). @jcape can you confirm?

Reply:
The motivation was originaly to make a utility that allows developers to find the block signing key more easily, but I am happy to move into the validator crate to where the business logic is actually taking place.